config/environments/development.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false #KIRSTIN for Docker
```

Comment:
Actually leaving this parameter true on my machine leads to the behavior, that the application tries to reach the database and redis locally. When I run docker they are no longer local in that sense. I do not know whether there is an other way to change this behavior, but changing this to false fixed this issue and made it possible for me to connect to the database. In production this parameter is false, because the default is false. 

I do not know which error messages you mean, because if I have an error, it does show me the stack trace on the web page.
Maybe there is a way to get good error messages again if they did get worse??